src/sys/event.rs

```diff
@@ -79,16 +78,25 @@ pub enum EventFilter {
 }
 
 #[cfg(any(target_os = "macos", target_os = "ios",
```

Comment:
Please alphabetize the order of the OSes since you're already modifying this attribute.